test/server.js

```diff
@@ -13,9 +13,43 @@ const config = {
     port: 5432,
     name: 'sensorthingstest',
     user: 'postgres',
-    pass: '12345678'
+    pass: '12345678',
+    hooks: {
+      'beforeValidate': instance => {
+        instance.beforeValidate = 'beforeValidate';
+      },
+      'afterValidate': instance => {
+        instance.afterValidate = 'afterValidate';
+      },
+      'beforeCreate': instance => {
+        instance.beforeCreate = 'beforeCreate';
+      },
+      'afterCreate': instance => {
+        instance.afterCreate = 'afterCreate';
+      },
+      // From Sequelize documentation:
+      // (http://docs.sequelizejs.com/en/latest/docs/hooks/)
+      // "The only way to call beforeDestroy/afterDestroy hooks are on
+      // associations with `onDelete: 'cascade'` and the option `hooks: true`"
+      // TBD It seems our ST API implementation does not exercise these hooks.
+      // Perhaps we should not implement them.
+/*
```

Comment:
The Sequelize documentation makes me think the `beforeDestroy` and `afterDestroy` hooks will not be run during the use of our ST API. Is that correct? If so, should we not allow clients to set them?